server/src/main/java/org/elasticsearch/cluster/coordination/StableMasterHealthIndicatorService.java

```diff
@@ -43,7 +43,7 @@ public class StableMasterHealthIndicatorService implements HealthIndicatorServic
     public static final String GET_HELP_GUIDE = "https://ela.st/getting-help";
     public static final Diagnosis CONTACT_SUPPORT_USER_ACTION = new Diagnosis(
         new Diagnosis.Definition(
-            "contact_support",
+            "elasticsearch:health:" + NAME + ":diagnosis:contact_support",
```

Comment:
Could we make `"elasticsearch:health:"` a public static variable somewhere so that we don't have to repeat it over and over? We risk introducing a typo that'd be easy to miss with how often it's repeated